src/bookshelf/rebar.lock

```diff
@@ -1,11 +1,15 @@
 [{<<"envy">>,
-  {git,"git://github.com/manderson26/envy.git",
-       {ref,"954c87adcfea892db462f56059eb8bc2857ab50b"}},
-  1},
+  {git,"https://github.com/manderson26/envy",
+       {ref,"8ba233bae37d87c319f3b4c5ae6973576986b36c"}},
+  0},
```

Comment:
Just a curiosity question here, is the `0` (last tuple element) here a reference to the depth of the node in the dependency tree?